AMW_angular/io/src/app/apps/app-add/app-add.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, EventEmitter, Input, Output, Signal } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { Release } from '../../settings/releases/release';
+import { Release as Rel } from '../../resource/release';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { Resource } from '../../resource/resource';
+import { AppCreate } from '../app-create';
+
+@Component({
+  selector: 'app-app-add',
+  standalone: true,
+  imports: [FormsModule, NgSelectModule],
+  templateUrl: './app-add.component.html',
+})
+export class AppAddComponent {
+  @Input() releases: Signal<Release[]>;
+  @Input() appServerGroups: Signal<Resource[]>;
+  @Output() saveApp: EventEmitter<AppCreate> = new EventEmitter<AppCreate>();
```

Comment:
We could use `releases = input.required<Release[]>()` and `saveApp = output<AppCreate>()` instead of `@Input` and `@Output`